src/analysis/normed_space/basic.lean

```diff
@@ -491,7 +541,7 @@ lemma mul_right_bound {α : Type*} [normed_ring α] (x : α) :
 λ y, by {rw mul_comm, convert norm_mul_le y x}
 
 /-- Normed ring structure on the product of two normed rings, using the sup norm. -/
-instance prod.normed_ring [normed_ring α] [normed_ring β] : normed_ring (α × β) :=
+instance prod.normed_ring [normed_ring β] : normed_ring (α × β) :=
 { norm_mul := assume x y,
```

Comment:
Also put a `norm_one_class` structure on the product of two rings with `norm_one_class`?